client/src/components/App.jsx

```diff
@@ -70,22 +60,24 @@ class App extends Component {
     }
   }
 
+  
   render() {
     const menu = (
       <div id="menu" style={{ fontSize: "28px", fontWeight: "bold" }} onClick={this.toggleMenu}>
         Menu
       </div>
     );
-    const { currentUser, tags, allChats } = this.state;
+    const { currentUser, allChats } = this.state;
     return (
       <div>
         <Router>
           <Nav currentUser={currentUser} toggleMenu={this.toggleMenu} />
           <Switch>
             <Route exact={true} path="/" render={() => <Login />} />
             <Route
+              exact={true}
```

Comment:
you can use the exact attribute here, but it is mostly meant for the render of the page whose endpoint is "/"